nibabel/freesurfer/tests/test_io.py

```diff
@@ -92,18 +91,19 @@ def test_geometry():
         with clear_and_catch_warnings() as w:
             warnings.filterwarnings('always', category=DeprecationWarning)
             read_geometry(surf_path, read_metadata=True)
-        assert_true(any('volume information contained' in str(ww.message)
+        assert(any('volume information contained' in str(ww.message)
```

Comment:
Can you do the same for the next few `assert_true(any(...))`?

```suggestion
        assert any('volume information contained' in str(ww.message) for ww in w)
```